docker-connector-product/README.md

```diff
@@ -1,15 +1,16 @@
 # Docker Connector
 
-YOUR DESCRIPTION GOES HERE
+Create, start, stop, remove Docker containers with the Axon Ivy's Docker Connector directly from your business processes.
 
 ## Demo
 
-YOUR DEMO DESCRIPTION GOES HERE
+The demo process creates and starts an additional Axon Ivy Engine in a Docker container. 
+Then the process shows a HTML Dialog where you can open an additional browser tab that shows the info page of the Axon Ivy Engine.
+If you proceed the HTML Dialog then the container gets stopped and removed.
```

Comment:
this sentence is broken: maybe this one fits?
If you proceed in the Html Dialog, the container gets stopped and remove.